src/main/java/com/fasterxml/jackson/databind/util/EnumValues.java

```diff
@@ -267,6 +241,22 @@ protected static Enum<?>[] _enumConstants(Class<?> enumCls) {
         }
         return enumValues;
     }
+
+    protected static String _findNameToUse(String explicitName, String otherName, boolean toLowerCase) {
```

Comment:
I questioned for a second whether to lazy-initialize `otherName` param by passing a functional interface or something, but thought it would be overkill to start with.

Reply:
Agreed, I think it's ok to eagerly process things in this case. Should not be on critical path.